sdk/iot/pnp/src/az_iot_pnp_client_telemetry.c

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <stdint.h>
+
+#include "az_iot_pnp_client.h"
+#include <az_precondition.h>
+#include <az_precondition_internal.h>
+#include <az_result.h>
+#include <az_span.h>
+
+#include <_az_cfg.h>
+
+static const uint8_t pnp_telemetry_param_separator = '&';
+static const uint8_t pnp_telemetry_param_equals = '=';
+
+static const az_span pnp_telemetry_component_name_param = AZ_SPAN_LITERAL_FROM_STR("%24.ifname");
+static const az_span pnp_telemetry_content_type_param = AZ_SPAN_LITERAL_FROM_STR("%24.ct");
+static const az_span pnp_telemetry_content_encoding_param = AZ_SPAN_LITERAL_FROM_STR("%24.ce");
+
+static az_result az_add_telemetry_property(
```

Comment:
I think there is guidance against prefixing static functions with `az_`. Only the public ones should.
https://azure.github.io/azure-sdk/clang_design.html#functions

Reply:
Fixed.